.github/styles/Google/MarkdownLinks.yml

```diff
@@ -0,0 +1,7 @@
+extends: conditional
+message: "Remember to include the link."
+level: error
+ignorecase: false
+# Ensures that the existence of 'first' implies the existence of 'second'.
+first: '(?:\[[\w-]+\])+(\[[\w-]+\])'
+second: '(\[[\w-]+\])+(?:: [\w/:?-]+$)'
```

Comment:
@jacobprall I noticed many of our Markdown reference-style links are broken because there's actually no reference, or the reference text doesn't match.

I _tried_ to write a rule to catch those, but regex is not my strong suit, and I found the Vale docs on conditional rules particularly unenlightening. Does this actually do what I want it to do?

Reply:
Looking at it again I think it should say `second: '(\[[\w-]+\])+(?:: [\w/:?=#-]+$)'` (added `=` and `#` to the url bit)

Reply:
Try running your regex through https://regex101.com/ and testing there - I think you might need to add an escape on one of the delimiters in the second line if I'm not mistaken. (I'm a little rusty on regex) first line looks good tho

Reply:
That site is super helpful!